docs/root/configuration/listeners/network_filters/_include/wasm-filter-proxy.yaml

```
@@ -0,0 +1,39 @@
+static_resources:
+  listeners:
+  - name: main
+    address:
+      socket_address:
+        address: 127.0.0.1
+        port_value: 10000
+    filter_chains:
+    - filters:
+      - name: envoy.filters.network.wasm
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.wasm.v3.Wasm
+          config:
+            name: "my_plugin"
+            vm_config:
+              runtime: "envoy.wasm.runtime.v8"
+              code:
+                local:
+                  filename: "lib/envoy_filter_http_wasm_example.wasm"
+              allow_precompiled: true
+      - name: envoy.filters.network.tcp_proxy
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.tcp_proxy.v3.TcpProxy
+          stat_prefix: tcp
+          cluster: tcp
+  clusters:
+    - name: tcp
+      type: strict_dns
```

altho this isnt strictly enforced by the server - the spec says that this should be all caps

```suggestion
      type: STRICT_DNS
```

---

@phlax I've tried to start Envoy with `stat_prefix` for the filter but it showed that this field was not supported.  So I thought this was the problem.